packages/sui-js/README.md

```diff
@@ -14,3 +14,25 @@ const {isMobile, osName} = stats(userAgent)
 domain.config('isMobile', isMobile) // bool
 domain.config('osName', osName) // string
 ```
+
+
+## cookie
+Parse, get and set cookies. Returns an object `cookie` with `parse`, `get` and `set` methods.
+
+**Note:** `set` method does not work on server side.
+
+```js
+import cookie from '@s-ui/js/lib/cookie'
+
+// Parse
+const {parse} = cookie
+domain.config('cookie', parse(cookies))
```

Comment:
I guess what you need is to provide all cookies to the domain as an object.
Could you do that with js-cookie package using `Cookies.get()` ?

Or do you really need to actually parse the cookie hash somewhere ?

Reply:
looks like `Cookies.get()` could work fine, thanks for point it out!
let me make sure it works properly in server side. if so, we'll be able to get rid of `cookie` package and only use `jsCookie`. i'll keep you posted.

Reply:
@davidbarna unfortunatelly, `jsCookie` does not work in server side. so we need to load these two libraries.